src/main/org/nlogo/compiler/CompilerMain.scala

```diff
@@ -29,7 +29,7 @@ private object CompilerMain {
     for(procedure <- structureResults.procedures.values.asScala) {
       procedure.topLevel = subprogram
       val tokens =
-        new IdentifierParser(program, oldProcedures, structureResults.procedures, false)
+        new IdentifierParser(program, oldProcedures, structureResults.procedures, forgiving = true)
```

Comment:
After this change, is there any time we want to run this with `forgiving = false`? If not, let's remove the argument.